lib/config/index.js

```diff
@@ -97,21 +98,33 @@ config.isGitLabEnable = config.gitlab.clientID && config.gitlab.clientSecret
 config.isMattermostEnable = config.mattermost.clientID && config.mattermost.clientSecret
 config.isLDAPEnable = config.ldap.url
 config.isSAMLEnable = config.saml.idpSsoUrl
-config.isPDFExportEnable = config.allowpdfexport
+config.isPDFExportEnable = config.allowPDFExport
 
 // merge legacy values
-if (config.imageUploadType && !config.imageuploadtype) {
-  config.imageuploadtype = config.imageUploadType
+let keys = Object.keys(config)
+const uppercase = /[A-Z]/
+for (let i = keys.length; i--;) {
+  let lowercaseKey = keys[i].toLowerCase()
+  // if the config contains uppercase letters
+  // and a lowercase version of this setting exists
+  // and the original, new, config with uppercase is not set
```

Comment:
this explanation is hard to read. I would expect this to read:
`and the correct config with uppercase is not set`

Reason: the "new" will not make sense to anyone but us. It's also weird that original and new refer to the same thing.